designer/server/src/test/scala/pl/touk/nussknacker/ui/integration/DictsFlowTest.scala

```diff
@@ -32,9 +32,43 @@ class DictsFlowTest
   private val EndNodeId      = "end"
   private val Key            = "foo"
   private val Label          = "Foo"
+  private val DictId         = "dict"
 
   override def nuTestConfig: Config = ConfigWithScalaVersion.TestsConfigWithEmbeddedEngine
 
+  test("query dict entries by label pattern") {
+    val response1 = httpClient.send(
+      quickRequest
+        .get(uri"$nuDesignerHttpAddress/api/processDefinitionData/$Streaming/dict/$DictId/entry?label=fo")
```

Comment:
I know that it isn't your orignal code, but let's change the Label to sth different. Currently, it could be confusing, because label looks like the key.